testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcSamlAttributeConsumingServiceIndexTest.java

```diff
@@ -14,7 +14,9 @@
 import org.junit.Assert;
 import org.junit.Test;
 import org.w3c.dom.Document;
+import org.w3c.dom.Element;
```

Comment:
nitpick: Changes in this file seem redundant.

Reply:
If you're ok with that, I can fix it in the next PR I have prepared already, i.e. metadata support for "SAML Attribute to Role" mapper.